### PR TITLE
Container Scanning: Documentation

### PR DESCRIPTION
**Pull request by meghfossa**
# Overview

Pretty Link: https://github.com/fossas/fossa-cli/blob/feat/complete-container-scanning-docs/docs/references/experimental/container/experimenal-scanner.md

This adds working documentation for experimental-scanner. 

This is sibling PR of https://github.com/fossas/fossa-cli/pull/1026. 

## Checklist

- [ ] I added tests for this PR's change (or explained in the PR description why tests don't make sense).
- [ ] If this PR introduced a user-visible change, I added documentation into `docs/`.
- [ ] If this change is externally visible, I updated `Changelog.md`. If this PR did not mark a release, I added my changes into an `# Unreleased` section at the top.
- [ ] If I made changes to `.fossa.yml` or `fossa-deps.{json.yml}`, I updated `docs/references/files/*.schema.json`. You may also need to update these if you have added/removed new dependency type (e.g. `pip`) or analysis target type (e.g. `poetry`).

### CODE REVIEW

**Review by spatten — APPROVED** [review]
I made a bunch of comments, but I don't think anything is blocking

File: docs/references/experimental/container/experimenal-scanner.md [hunk 1]
```diff
@@ -1,66 +1,343 @@
...
+| .Net (packet, projectjson, etc.)     | :white_check_mark: | [.Net](./../../strategies/languages/dotnet/README.md)            |
+| Perl                                 | :white_check_mark: | [Perl](./../../strategies/languages/perl/perl.md)                |
+| Swift (xcode, swift package manager) | :white_check_mark: | [Swift](./../../strategies/platforms/ios/swift.md)               |
+| Carthage                             | :white_check_mark: | [Carthage](./../../strategies/platforms/ios/carthage.md)         |
+| Fortran (fpm)                        | :white_check_mark: | [Fortran](./../../strategies/languages/fortran/fortran.md)       |
+| Cocoapods                            | :warning:          | [CocoaPods](./../../strategies/platforms/ios/cocoapods.md)       |
+| Nim (nimble)                         | :warning:          | [Nim](./../../strategies/languages/nim/nimble.md)                |
+| Dart (pub)                           | :warning:          | [Dart](./../../strategies/languages/dart/pub.md)                 |
+| Maven                                | :warning:          | [Maven](./../../strategies/languages/maven/maven.md)             |
```
[the remark this answers is not in the document]
Comment (jssblck) [reply docs/references/experimental/container/experimenal-scanner.md:247]: These are updated as part of https://github.com/fossas/fossa-cli/pull/1026

File: docs/references/strategies/system/apk/apk.md [hunk 2]
```diff
@@ -0,0 +1,54 @@
+# APK
+
+The Alpine Package Manager (apk).
+
+> This analysis is only ran for Container Scanning.
+
+## Discovery
+
+Find file named `installed` under `**/apk/**`.
```
[the remark this answers is not in the document]
Comment (jssblck) [reply docs/references/strategies/system/apk/apk.md:9]: The image root. Realistically the strategy just looks a path ending in `installed` that contains `apk/` in the path: https://github.com/fossas/fossa-cli/blob/fc517bd0aedbbad731703f7bfd1b946a8bdcd419/src/Strategy/ApkDatabase.hs#L65

File: docs/references/subcommands/container.md [hunk 3]
```diff
@@ -1,62 +1,106 @@
...
 For example:
 
 ```bash
 fossa container test redis:alpine
 ```
 
+## `fossa container list-targets <ARG>`
+
+Lists all detected analysis targets in an image.
+This output can be useful to understand what is going to be analyzed via `fossa container analyze <ARG> --experimental-scanner`,
```
[the remark this answers is not in the document]
Comment (jssblck) [reply docs/references/subcommands/container.md:51]: Good call, moved.

**Pull request merged**